apps/vscode/command_client/command_client.py

```diff
@@ -13,10 +13,10 @@
 # to remove it
 STALE_TIMEOUT_MS = 60_000
 
-# The amount of time to wait for VSCode to perform a command, in seconds
-VSCODE_COMMAND_TIMEOUT_SECONDS = 3.0
+# The amount of time to wait for client application to perform a command, in seconds
```

Comment:
We should probably standardise on terminology here. I've been referring to VSCode as the server, not the client. Maybe "target application"? Cc/ @rntz @phillco

Reply:
Just "the application" seems fine here.

Reply:
Removed client